src/Knp/DictionaryBundle/Dictionary/CallableDictionary.php

```diff
@@ -116,11 +120,9 @@ public function getIterator()
      */
     public function serialize()
     {
-        $this->hydrate();
-
         return serialize([
             'name'   => $this->name,
-            'values' => $this->values,
+            'values' => $this->values ?: array(),
```

Comment:
Since we aim to drop unmainted php versions, you can use shorter syntax for the array